docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -97,105 +269,149 @@ pullSecret: |
-  ${CLUSTER_ID}/master.ign | \
-  awk -F ',' '{ print $2 }' | \
-  base64 --decode)
+=== Terraform Cluster Config
+
+[NOTE]
+====
+Check https://syn.tools/commodore/running-commodore.html[Running Commodore] for details on how to run commodore.
+====
+
+. Prepare Commodore inventory.
++
+[source,console]
+----
+mkdir -p inventory/classes/
+git clone $(curl -sH"Authorization: Bearer ${COMMODORE_API_TOKEN}" "${COMMODORE_API_URL}/tenants/${TENANT_ID}" | jq -r '.gitRepo.url') inventory/classes/${TENANT_ID}
 ----
 
-. Prepare RHCOS template
+. Prepare Terraform cluster config
 +
 [source,console]
 ----
-RHCOS_VERSION="4.7.7"
+CA_CERT=$(jq -r '.ignition.security.tls.certificateAuthorities[0].source' \
+  "${INSTALLER_DIR}/master.ign" | \
+  awk -F ',' '{ print $2 }' | \
+  base64 --decode)
 
-curl "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/${RHCOS_VERSION}/rhcos-${RHCOS_VERSION}-x86_64-openstack.x86_64.qcow2.gz" | gunzip > rhcos-${RHCOS_VERSION}.qcow2
+pushd "inventory/classes/${TENANT_ID}/"
 
-virt-edit -a rhcos-${RHCOS_VERSION}.qcow2 \
-  -m /dev/sda3:/ /loader/entries/ostree-1-rhcos.conf \
-  -e 's/openstack/exoscale/'
+yq eval -i '.applications += ["openshift4-terraform"]' ${CLUSTER_ID}.yml
 
-exo sos upload "${CLUSTER_ID}-bootstrap" rhcos-${RHCOS_VERSION}.qcow2
-exo sos acl add "${CLUSTER_ID}-bootstrap" rhcos-${RHCOS_VERSION}.qcow2 --public-read
+yq eval -i ".parameters.openshift.infraID = \"$(jq -r .infraID "${INSTALLER_DIR}/metadata.json")\"" \
+  ${CLUSTER_ID}.yml
 
-exo vm template register "rhcos-${RHCOS_VERSION}" \
-  --checksum $(md5sum rhcos-${RHCOS_VERSION}.qcow2 | awk '{ print $1 }') \
-  --boot-mode uefi \
-  --disable-password \
-  --username core \
-  --description "Red Hat Enterprise Linux CoreOS (RHCOS) ${RHCOS_VERSION}" \
-  --url "https://${AWS_S3_ENDPOINT}/${CLUSTER_ID}-bootstrap/rhcos-${RHCOS_VERSION}.qcow2"
+yq eval -i ".parameters.openshift.clusterID = \"$(jq -r .clusterID "${INSTALLER_DIR}/metadata.json")\"" \
+  ${CLUSTER_ID}.yml
 
-exo sos remove "${CLUSTER_ID}-bootstrap" rhcos-${RHCOS_VERSION}.qcow2
+yq eval -i ".parameters.openshift.appsDomain = \"apps.${CLUSTER_ID}.${BASE_DOMAIN}\"" \
+  ${CLUSTER_ID}.yml
 
-export TF_VAR_rhcos_template="rhcos-${RHCOS_VERSION}"
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.base_domain = \"${BASE_DOMAIN}\"" \
+  ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.ignition_ca = \"${CA_CERT}\"" \
+  ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.rhcos_template = \"${TF_VAR_rhcos_template}\"" \
+  ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.bootstrap_bucket = \"${BOOTSTRAP_BUCKET}\"" \
+  ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.ssh_key = \"$(cut -d' ' -f1,2 <${SSH_PUBLIC_KEY})\"" \
+  ${CLUSTER_ID}.yml
```

Comment:
Might be fine to just have `$(cat $SSH_PUBLIC_KEY)` now that we control the comment